adminSiteServer/apiRouter.ts

```diff
@@ -2468,6 +2470,15 @@ deleteRouteWithRWTransaction(apiRouter, "/gdocs/:id", async (req, res, trx) => {
     if (gdoc.published && checkIsGdocPostExcludingFragments(gdoc)) {
         await removeIndividualGdocPostFromIndex(gdoc)
     }
+    // Assets have TTL of one week in Cloudflare. Add a redirect to make sure
+    // the page is no longer accessible.
+    // https://developers.cloudflare.com/pages/configuration/serving-pages/#asset-retention
+    const ttl = dayjs().add(8, "days").toDate()
+    await db.knexRawInsert(
+        trx,
+        `INSERT INTO redirects (source, target, ttl) VALUES (?, ?, ?)`,
+        [getCanonicalUrl("", gdoc), "/", ttl]
```

Comment:
I _slightly_ prefer the timestamp computation to also be happening within MySQL syntax.
That way, there's no way for there to be weird timezone or other issues between the two. And thankfully, the MySQL syntax for this is still pretty readable.

```suggestion
        `INSERT INTO redirects (source, target, ttl) VALUES (?, ?, DATE_ADD(NOW(), INTERVAL 8 DAY) )`,
        [getCanonicalUrl("", gdoc), "/"]
```

Reply:
Good idea! Changed.